src/hooks/useAppData.tsx

```diff
@@ -49,12 +49,18 @@ export const AppDataProvider = ({ children }: PropsWithChildren) => {
       }
     }
 
-    const task = {
-      count:
-        (appState?.appState.notifications?.forms || 0) +
-        (appState?.appState.notifications?.billing || 0) +
-        (appState?.appState.notifications?.contract || 0),
-    }
+    let count = 0
+    appState?.appState.settings?.notifications?.map((el) => {
+      if (el.show) {
+        if (appState?.appState.notifications) {
+          count +=
+            appState?.appState.notifications[el.key as keyof INotification]
+        }
```

Comment:
This might lead to an edge case where `appState?.appState.notifications` is empty array [], however an empty array is still truthy in Javascript.

Can you please confirm if this will cause issues

Reply:
I think after the OUT-197 PR we should throw an error if notifications is going to be set to empty array. Please confirm again though.